modules/frontend/combiner/trace_by_id_v2_test.go

```diff
@@ -36,17 +36,9 @@ func TestNewTraceByIdV2ReturnsAPartialTrace(t *testing.T) {
 		Trace:   test.MakeTrace(2, []byte{0x01, 0x02}),
 		Metrics: &tempopb.TraceByIDMetrics{},
 	}
-	resBytes, err := proto.Marshal(traceResponse)
-	require.NoError(t, err)
-	response := http.Response{
-		StatusCode: 200,
-		Header: map[string][]string{
-			"Content-Type": {"application/protobuf"},
-		},
-		Body: io.NopCloser(bytes.NewReader(resBytes)),
-	}
+
 	combiner := NewTraceByIDV2(10, api.HeaderAcceptJSON)
```

Comment:
Is this test working as expected?  The combiner has a limit of 10 bytes, but we only input 2 bytes, but it still returns partial status.